de_templates/test_config_ingest.yaml

```diff
@@ -0,0 +1,38 @@
+pipeline_name: Ingest_AzureSql_Example
+pipeline_description: "Ingest from demo Azure SQL database using ingest config file."
+data_source_type: azure_sql
+
+# Used to create an ADF dataset and a linked service, named ds_<dataset_name> and ls_<dataset_name>
+dataset_name: AzureSql_Example
+
+key_vault_linked_service_name: ls_KeyVault
+data_source_password_key_vault_secret_name: sql-password
+
+# Optional parameter - if not set, the default is Incremental
+default_arm_deployment_mode: Incremental
+
+
+# Azure DevOps configurations
+
+ado_variable_groups_nonprod:
+  - amido-stacks-de-pipeline-nonprod
+  - amido-stacks-infra-credentials-nonprod
+  - stacks-credentials-nonprod-kv
+
+ado_variable_groups_prod:
+  - amido-stacks-de-pipeline-prod
+  - amido-stacks-infra-credentials-prod
+  - stacks-credentials-prod-kv
+
+
+# Datalake containers
+
+bronze_container: raw
+silver_container: staging
+gold_container: curated
+
+
+# Workload config
+
+window_start_default: 2020-01-01
+window_end_default: 2020-01-31
```

Comment:
These defaults will only really be used when testing / debugging the pipeline in ADF (when running properly you'd be passing these values) - so would be tempted to leave them out of the config here as it might cause confusion?

Reply:
... I now see this also gets used to in the tests template. So in which case, maybe add a comment here that these are default dates for tests.

Reply:
Updated. I added a comment and made these values optional. If you want to, I can also remove them altogether.